rpm/containers-common.spec

```diff
@@ -0,0 +1,222 @@
+# Below definitions are used to deliver config files from a particular branch
+# of c/image, c/storage  and c/shortnames vendored in all of Buildah, Podman and Skopeo.
+# These vendored components must have the same version. If it is not the case,
+# pick the oldest version on c/image, c/storage and c/shortnames vendored in
+# Buildah/Podman/Skopeo.
+
+# Replace `main` below with upstream version tags before cutting a new release of c/common
+%global image_branch main
+%global storage_branch main
+%global shortnames_branch main
```

Comment:
Who is going to do that? I think we have the issue today that many config files do not actually match the tagged versions 1 to 1.

go.mod should already specify the correct version so can we use this to automate this somehow?

Reply:
added a packit action that will run `rpm/update-lib-versions.sh` on downstream PRs. We can only test this once downstream PRs are created though.